src/runtime/virtcontainers/network_linux.go

```diff
@@ -1613,8 +1613,34 @@ func removeTxRateLimiter(endpoint Endpoint, networkNSPath string) error {
 func validGuestRoute(route netlink.Route) bool {
 	return route.Protocol != unix.RTPROT_KERNEL
 }
+func validGuestNeighbor(neigh netlink.Neigh, gwSet map[string]struct{}) bool {
```

Comment:
[nitpick] Consider using more descriptive names for your variables. It is not explicit what the acronym "gw" means in the context of this function.